test/e2e/discord_driver_test.go

```diff
@@ -257,6 +274,8 @@ func (d *discordTester) WaitForMessagePostedWithAttachment(userID, channelID str
 
 	var fetchedMessages []*discordgo.Message
 	var lastErr error
+	var diffMessage string
+	common := -1
```

Comment:
Can we revert this change to the previous form? I think the behavior when initial `common` is 0 is better, as we won't print a diff when there are no common blocks. What do you think?

```suggestion
	var common int
```

Reply:
@pkosiec actually the original code is what you suggested. The reason I changed this in my last commit is there is only one line in the message of `AttachmentAssertion`. If there is any difference, the common will be always be `0`, then no diff will be print out. What do you think?

Reply:
If we don't want print diff there is only one message, feel free to apply the suggestions and merge the PR.

Reply:
Thank you @wangyangjun, very good point. I think I found a solution - check the message limits. See this commit and tell me if that looks OK for you 👍 
https://github.com/kubeshop/botkube/pull/811/commits/e5300cf02db8cb7040d7c364574fd488fa623786

Reply:
@pkosiec Yes, it looks good to me.

Reply:
Cool, I've just pushed the same changes for Discord because I forgot to do it.. Enabling auto-merge then!